allmodules

```diff
@@ -12,6 +12,7 @@
 
 # module-directory           released
 .                            yes
+docstore/mongodocstore       no
```

Comment:
I believe this should be "yes". This information is only used by the release tool, and this module will be tagged

Reply:
Done.